builtin/game/chat.lua

```diff
@@ -66,8 +66,15 @@ core.register_on_chat_message(function(name, message)
 	local has_privs, missing_privs = core.check_player_privs(name, cmd_def.privs)
 	if has_privs then
 		core.set_last_run_mod(cmd_def.mod_origin)
-		local _, result = cmd_def.func(name, param)
-		if result then
+		local success, result = cmd_def.func(name, param)
+		if success == false and result == nil then
+			core.chat_send_player(name, "-!- Invalid command usage")
+			local help_def = core.registered_chatcommands["help"]
```

Comment:
Servers might remove the help command and implement something on their own. I think a `nil` check for `help_def` would be appropriate here.

Reply:
I've added a check.